sensor_msgs/PointCloud2 input_pc
---
geometry_msgs/Point object_centroid